File: source/code/vbnet/authentication/generate_oauth_token_req.vb
Dim grantRequest = OauthRequests.NewPasswordGrantRequest() _
    .SetLogin("han@newrepublic.gov") _
    .SetPassword("first2shoot!") _
    .Build()

Dim grantResponse = Await app.NewPasswordGrantAuthenticator() _
    .AuthenticateAsync(grantRequest)

File: source/code/java/authentication/generate_oauth_token_req.java
OAuthPasswordGrantRequestAuthentication request =
    OAuthRequests.OAUTH_PASSWORD_GRANT_REQUEST
        .builder()
        .setPassword("first2shoot!")
        .setLogin("han@newrepublic.gov")
        .build();

OAuthGrantRequestAuthenticationResult authenticationResult =
    Authenticators.OAUTH_PASSWORD_GRANT_REQUEST_AUTHENTICATOR
        .forApplication(application)
        .authenticate(request);
